### PR TITLE
KAFKA-4058: Failure in org.apache.kafka.streams.integration.ResetIntegrationTest.testReprocessingFromScratchAfterReset

### PR DESCRIPTION
**Pull request by mjsax**
see #1765

### CODE REVIEW

File: streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java [hunk 1]
```diff
@@ -225,6 +228,16 @@ private void cleanGlobal() {
         Assert.assertEquals(0, exitCode);
     }
 
+    private void waitUntilConsumerGroupGotClosed(final long sleep) {
+        int retryCounter = 0;
+        while (this.adminClient.describeConsumerGroup(APP_ID).get().size() != 0) {
+            if (++retryCounter > 5) {
+                throw new RuntimeException("Consumer group did not time out after " + (5 * sleep) + " ms.");
+            }
+            Utils.sleep(STREAMS_CONSUMER_TIMEOUT);
```
Comment (guozhangwang) [inline streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java]: Backoff time of 2 sec might be a bit too large here. How about 100ms?

Comment (guozhangwang) [inline streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java]: More generally, could we reuse the `TestUtils.waitForCondition` function?

**Comment by mjsax** [comment]
Updated @guozhangwang

**Comment by guozhangwang** [comment]
@mjsax The jenkins build seems not re-triggered, as the test is still failing. Could you close and re-open to re-trigger the test?

**Comment by mjsax** [comment]
Replace by #1767

**Pull request closed without merging**